test/utils/pickers/describeValue/testPickerOpenCloseLifeCycle.tsx

```diff
@@ -391,4 +394,47 @@ export const testPickerOpenCloseLifeCycle: DescribeValueTestSuite<PickerValidVal
       expect(onClose.callCount).to.equal(0);
     });
   });
+
+  testSkipIf(
+    !['date-range', 'time-range', 'date-time-range'].includes(pickerParams.type) ||
+      (pickerParams as any).fieldType !== 'single-input',
+  )('should return back to start range position after reopening a range picker', async () => {
+    const pickerType = pickerParams.type as PickerRangeComponentType;
+    // If transitions are disabled, the `onExited` event is not triggered
+    config.disabled = false;
```

Comment:
Had to enable Transitions for this test, otherwise the `onExited` callback is not triggered. 🙈